modules/elasticsearch/src/test/java/org/testcontainers/elasticsearch/ElasticsearchContainerTest.java

```diff
@@ -27,37 +18,49 @@
 import org.junit.After;
 import org.junit.Test;
 
+import java.io.IOException;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertThrows;
+
 public class ElasticsearchContainerTest {
 
     /**
      * Elasticsearch version which should be used for the Tests
      */
-    private static final String ELASTICSEARCH_VERSION = Version.CURRENT.toString();
+    private static final String ELASTICSEARCH_VERSION = "7.8.0";
```

Comment:
Ideally we should get this from the Gradle definition. Is that possible?

Reply:
Depending on the module, it's not always the case that there's a 1:1 mapping between driver and image versions, but if there is for Elasticsearch that's quite nice. It could help keep our tests up to date.

We do this for Selenium, for which there's historically been a very tight coupling between driver version and required image version.

For the Selenium module, the code that does it is here: https://github.com/testcontainers/testcontainers-java/blob/923472ff114d99b2ac0e199a86829879c24c8471/modules/selenium/src/main/java/org/testcontainers/containers/SeleniumUtils.java#L17

If you wanted to use this for inspiration (maybe change `SeleniumUtils` to a general-purpose class inside our `test-support` module), perhaps it could work. We'd only want to use this for Testcontainers' internal tests, though - i.e. deprecation of the default-args constructor still applies.

Reply:
Right. I think that this should come within another PR so we don't overcomplicate things.
BTW I realized lately that I should have commented in #2320 and not here :(